packages/components/src/components/ResizableBox/ResizableBox.stories.tsx

```diff
@@ -38,6 +38,8 @@ export const ResizableBox: StoryObj<ResizableBoxProps> = {
         height: 100,
         minHeight: 50,
         maxHeight: 300,
+        updateHeightOnWindowResize: false,
+        updateWidthOnWindowResize: false,
```

Comment:
Btw you could add these props also to `argTypes`

Reply:
Yes I could but do we need it? Storybook automatically infers the controls for props based on the component's propTypes or TypeScript definitions. I can see many components don't have all their props added both in `args` and `argTypes` objects..

Reply:
I've added anything missing to argTypes for ResizableBox in this PR, even unnecessary. But I think we should make clear if we want to do this for all the components or not, or when (and then to update them, of course).